src/getNarrative.js

```diff
@@ -43,6 +51,10 @@ const getNarrative = async (req, res) => {
   try {
     utils.log(`Fetching narrative ${fetchURL} and streaming to client for parsing`);
     const response = await fetch(fetchURL);
+    if (response.status === 404) {
+      return res.status(404).send("The requested narrative does not exist.");
+    }
+
```

Comment:
This might belong in a new commit since it's an added feature rather than a conversion from 500 -> 4XX.

Reply:
Looks good. Does this change auspice behavior at all? (I wouldn't think so, but haven't tested)

Reply:
I have only been testing Nextstrain.org and so far everything works as expected. Do you have any recommendations for testing Auspice, @jameshadfield ?

Reply:
If you're running (nextstrain.org's implementation of) auspice as part of testing nextstrain.org, then that should be enough. I'm not sure if you've just been using curl or similar.